lemur/notifications/messaging.py

```diff
@@ -0,0 +1,102 @@
+"""
+.. module: lemur.notifications.messaging
+    :platform: Unix
+
+    :copyright: (c) 2015 by Netflix Inc., see AUTHORS for more
+    :license: Apache, see LICENSE for more details.
+
+.. moduleauthor:: Kevin Glisson <kglisson@netflix.com>
+
+"""
+
+import arrow
+from flask import current_app
+
+from lemur import database, metrics
```

Comment:
How does this work?  Everything else uses `from lemur.extensions. import metrics`, otherwise this just a module.

Reply:
It works because lemur's __init__ imports those two modules. That being said they should be imported from `lemur.extensions` to avoid circular imports.